packages/excalidraw/components/App.tsx

```diff
@@ -7878,16 +7882,25 @@ class App extends React.Component<AppProps, AppState> {
               suggestedBindings: getSuggestedBindingsForArrows(
                 selectedElements,
                 this.scene.getNonDeletedElementsMap(),
-              ),
+              )
             });
           }
-
+          // keyTest:(event)=>
+          window.addEventListener('keydown', (event) => {
+            if (event.key === 'Tab' && event.altKey) {
```

Comment:
The `Tab` key is not captured by the browser as the OS takes the keystroke to change windows first. So this won't work.

Reply:
could you help me @mtolmacs i am trying but default windows change first and after alt key+tab is pressing i am not getting it how to resolve.
thanks.